tensorboard/plugins/hparams/BUILD

```diff
@@ -105,6 +105,23 @@ py_binary(
     ],
 )
 
+py_binary(
+    name = "hparams_ml_demo",
```

Comment:
I’d opted to keep the original, non-ML demo because it’s significantly
faster, which is nice when you just need to demonstrate dummy data to
test the plugin. Each session takes about a minute on my workstation
(W-2135), whereas the entire non-ML demo takes 16 seconds.

But you’re right that this should at least be the default, so I’ll go
ahead and rename the old one to “`hparams_minimal_demo`”, promoting this
one to “`hparams_demo`”.

Reply:
Is the original demo still much faster if we set the epochs to 1?  It seems more straightforward to just have a single demo (and to fully remove the confusingly non-ML-related one).